src/blueprint/operation.rs

```diff
@@ -11,6 +11,8 @@ pub struct OperationQuery {
     file: String,
 }
 
+// test
+
```

Comment:
@coderabbitai
Whats wrong with the validate_operations func in this file.
Is it correctly validating each and every query passed? How multiple queries are passed actually.
I didnt see validate_operations func being called anywhere.
Show the appropriate place to call that function.
Look carefully, explain. Thank you.